r3bsource/trloii/R3BTrloiiTpatReader.h

```diff
@@ -54,7 +54,7 @@ class R3BTrloiiTpatReader : public R3BReader
     }
 
     // Skip events with Tpat=0
-    void SetSkipEventsTpatZero() { fSkipTpatZero = kTRUE; }
+    void SetSkipEventsTpatZero(Bool_t option) { fSkipTpatZero = option; }
```

Comment:
Please, change this line for backward compatibility:
void SetSkipEventsTpatZero(Bool_t opt=kTRUE) { fSkipTpatZero = option; }

By default it is set to false, if one calls this function from the macros fSkipTpatZero is set to true, there are two ways to invoke it:

SetSkipEventsTpatZero();
or
SetSkipEventsTpatZero(true);

Reply:
Done.